src/module-rebuilder.ts

```diff
@@ -159,6 +165,24 @@ export class ModuleRebuilder {
     return fs.pathExists(path.resolve(this.modulePath, 'prebuilds', `${process.platform}-${this.rebuilder.arch}`, `electron-${this.rebuilder.ABI}.node`))
   }
 
+  private restoreEnv(env: any) {
```

Comment:
Might want to note in a comment that this complicated way to restore the environment is due to Windows.